### ISSUE TITLE
Feature Request: Fix Accessibility BoundingRects for TextRanges

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/terminal#2103]
# Summary of the new feature/enhancement

Thanks to #2083, we now have BoundingRects for text ranges. We can modify them and explore the text buffer properly.

Just one little problem...
![image](https://user-images.githubusercontent.com/11050425/61896035-e9c1df80-aec8-11e9-8578-16d14adb91bb.png)

The BoundingRects aren't actually on the text ranges properly. They update and function correctly, but they should look something like how they do in ConHost:
![image](https://user-images.githubusercontent.com/11050425/61896134-28579a00-aec9-11e9-8ff7-99552e54fa9c.png)

# Proposed technical implementation details (optional)
There's a few places in UiaTextRange that use `ScreenToClient()` and `ClientToScreen();` We can use them in ConHost because we have an HWND. Unfortunately, Windows Terminal doesn't have access to an HWND (and it seems like that'll be really hard to do).

XAML, on the other hand, should already know some information about bounding rects. That's how we got the one around the TermControl to appear and all of the other XAML elements.

We need to investigate if/how we can leverage XAML's knowledge of screen bounding rects and just pass them along to the UiaTextRange.
